src/Validator.php

```diff
@@ -226,7 +226,11 @@ private function getNumericRepresentation($letterRepresentation)
     private function local_bcmod($operand, $modulus)
     {
         if (function_exists('bcmod')) {
-            return bcmod($operand, $modulus);
+            $oldScale = bcscale(0);
+            $mod = bcmod($operand, $modulus);
+            bcscale($oldScale);
```

Comment:
it seems that this only works on PHP +7.3 as otherwise you would be passing `true` here: https://3v4l.org/tjCce

Reply:
Oof :(
This being the case, maybe we should use `ini_get` & `ini_set` for `bcmath.scale`?

Reply:
Yeah, was a mistake on my side. I added a direct version check now.